mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
+    })
+    .catch(error => { 
+      res.status(500).json({
+        error: errorHandler(error), // Used errorHandler for consistency
```

Comment:
The comment `// Used errorHandler for consistency` is not necessary and can be removed. The code is self-explanatory, and the comment does not add any additional value.